pkg/odo/cli/project/create.go

```diff
@@ -77,20 +77,24 @@ func (pco *ProjectCreateOptions) Run(cmd *cobra.Command) (err error) {
 		return err
 	}
 	s.End(true)
+
+	successMessage := fmt.Sprintf(`Project %q is ready for use`, pco.projectName)
 	log.Successf(successMessage)
```

Comment:
Optional: As smartly as `log.Successf` is written, can you add a tiny comment saying that `log.Successf`  will check for JSON before printing the message? I felt a little lost while reading the code.

Reply:
These calls are quite confusing, I agree. I removed some `if !log.IsJSON()` surrounding these calls in this PR, which were not necessary. Probably we should try to make the `log` package or the functions more self-explanatory